scripts/windows/setup/IotEdgeSecurityDaemon.ps1

```diff
@@ -921,7 +933,7 @@ function Install-Packages(
 
         if ((Test-MobyNeedsToBeMoved) -or (Test-LegacyInstaller)) {
             Write-HostRed
-            Write-HostRed ('IoT Edge is installed in an invalid location. ' + $ReinstallMessage)
+            Write-HostRed ('IoT Edge or the IoT Edge Moby Engine is installed in an invalid location. Probably there is an old preview install present and it needs to be uninstalled or the system should be re-imaged.' + $ReinstallMessage)
```

Comment:
Space is missing from the end of the string literal.

`re-imaged.' + $ReinstallMessage` -> `re-imaged. ' + $ReinstallMessage`

Same for the other instances of this.

Reply:
fixed